semgrep-core/Analyzing/Dataflow_constness.mli

```diff
@@ -1,7 +1,11 @@
 
-type mapping = AST_generic.constness Dataflow.mapping
+module G = AST_generic
```

Comment:
please no module aliasing in .mli (also semgrep for ocaml does not like it because it's not in
the current grammar for ocaml in pfff)

Reply:
Note that you can do 'make check' inside semgrep-core/ and it will run semgrep on the ocaml code.
(you need to have semgrep in your path, so you can do pipenv shell from semgrep/semgrep/ to get semgrep in your PATH).

Reply:
Fixed! Thanks for the tip, I still get problems to run Python's semgrep wrapper (it worked but then the setup changed and it no longer works). But I'll try to get it running.